docs/linux/program-type/BPF_PROG_TYPE_SCHED_ACT.md

```diff
@@ -4,5 +4,216 @@ description: "This page documents the 'BPF_PROG_TYPE_SCHED_ACT' eBPF program typ
 ---
 # Program type `BPF_PROG_TYPE_SCHED_ACT`
 
-!!! example "Docs could be improved"
-    This part of the docs is incomplete, contributions are very welcome
+<!-- [FEATURE_TAG](BPF_PROG_TYPE_SCHED_CLS) -->
+[:octicons-tag-24: v4.1](https://github.com/torvalds/linux/commit/94caee8c312d)
+<!-- [/FEATURE_TAG] -->
+
+This program type allows for the implementation of a Traffic Control (TC) action in eBPF.
+The details of the TC have been introduced in [Program type `BPF_PROG_TYPE_SCHED_CLS`](./BPF_PROG_TYPE_SCHED_CLS.md).
+
+## Usage
+
+TC Action programs are typically put into an [ELF](../../concepts/elf.md) section prefixed with `action/`. The TC Action program is called by the kernel with a [`__sk_buff`](../program-context/__sk_buff.md) context. The return value can be one of:
+
+* `TC_ACT_UNSPEC` (-1) - Signals that the default configured action should be taken.
+* `TC_ACT_OK` (0) - Signals that the packet should proceed.
+* `TC_ACT_RECLASSIFY` (1) - Signals that the packet has to re-start classification from the root qdisc. This is typically used after modifying the packet so its classification might have different results.
+* `TC_ACT_SHOT` (2) - Signals that the packet should be dropped, no other TC processing should happen.
+* `TC_ACT_PIPE`	(3) - While defined, this action should not be used and holds no particular meaning for eBPF classifiers.
+* `TC_ACT_STOLEN` (4) - While defined, this action should not be used and holds no particular meaning for eBPF classifiers.
+* `TC_ACT_QUEUED` (5) - While defined, this action should not be used and holds no particular meaning for eBPF classifiers.
+* `TC_ACT_REPEAT` (6) - While defined, this action should not be used and holds no particular meaning for eBPF classifiers.
+* `TC_ACT_REDIRECT`	(7) - Signals that the packet should be redirected, the details of how and where to are set as side effects by [helpers functions](../helper-function/index.md).
```

Comment:
It seems that unlike for CLS programs, ACT programs do support `TC_ACT_PIPE`. 
See https://elixir.bootlin.com/linux/v6.12.1/source/net/sched/act_bpf.c#L72

According to the [man page](https://man7.org/linux/man-pages/man8/tc-bpf.8.html) its 

> TC_ACT_PIPE (3) , will iterate to the next action, if available